### PR TITLE
Add Sendgrid-powered email list subscription

### PR DESCRIPTION
**Pull request by elliotwhitehead**
## Description
A few sentences describing the overall goals of the pull request's commits.

## Related Issues
Link to any issues that are relevant to this PR. This could be an issue this PR fixes.

## Some questions
- [x] I read the contributing guidelines
- [ ] I'm adding a new event/footer link
- [ ] I'm editing an existing event/footer link
- [x] I'm adding a new feature/fixing a bug

If you answered No to question 1, go back and read the [instructions](.github/CONTRIBUTING.md) carefully.

## Additional Notes
Do you want to add anything else? We :heart: to hear your opinions!

### CODE REVIEW

**Comment by elliotwhitehead** [comment]
It's finally done! Complete with Success / failure messages!

image

image

File: index.md [hunk 1]
```diff
@@ -30,7 +30,25 @@ layout: base
...
+                        <input id="email-input" type="email" class="form-control" name="email" placeholder="ralphie@colorado.edu">                        
+                        <span class="input-group-btn">
+                            <button id="email-submit" class="btn btn-success" type="submit"><i class="fa fa-paper-plane"></i></button>
+                        </span>
+                    </div>
+                    <div id="email-success" class="notice-message">
+                        <p>Thanks for subscribing! We're excited to see you soon!</p>
+                    </div>
+                    <div id="email-failure" class="notice-message">
+                        <p>Hmmm... something went wrong. Try again, and <a href="mailto:info@hackcu.org">let us know</a> if this persists..</p>
```
Comment (AparaV) [inline index.md]: Do we have an `info@hackcu.org` email?

**Review by AparaV — APPROVED** [review]
This looks cool! Feel free to merge it.

File: index.md (hunk 1 above)
Comment (casassg) [reply index.md]: No, we don't have such email heh. Change to contact@hackcu.org pls

**Review by casassg — CHANGES_REQUESTED** [review]
Email not existing XD

**Comment by elliotwhitehead** [comment]
Haha, good catch @AparaV. I'll get that fixed. CC @casassg

**Pull request merged**